packages/app-builder/public/locales/en/transfercheck.json

```
@@ -1,3 +1,11 @@
 {
-  "transfer_detail": "Transfer details"
+  "transfer.search.placeholder": "Search transfer by id",
+  "transfer.search.empty": "No transfer found",
+  "transfer_detail.status.title": "Transfer status",
+  "transfer_detail.status.neutral": "Neutral",
+  "transfer_detail.status.suspected_fraud": "Suspected fraud",
+  "transfer_detail.status.confirmed_fraud": "Confirmed fraud",
+  "transfer_detail.title": "Transfer details",
+  "transfer_detail.transfer_data.title": "Transfer data",
+  "transfer_detail.transfer_data.description": "Here is the transfer id associated to the source information system <TransferIdLabel>transfer_id</TransferIdLabel>: <TransferIdValue>{{transferId}}</TransferIdValue>"
```

```suggestion
  "transfer_detail.transfer_data.description": "Here is the Marble transfer id associated to the client information system <TransferIdLabel>transfer_id</TransferIdLabel>: <TransferIdValue>{{transferId}}</TransferIdValue>"
```
? (if I understand correctly)

---

No, that's the partner trasnfer ID comming from its information system.
I just wanted to display it from our trasnfer detail page, so it's possible for our partner users to easilly copy-paste the id to find in their internal tools more information.

It may be useless though

---

all good 👍